code/modules/projectiles/guns/ballistic/rifle.dm

```diff
@@ -3,6 +3,9 @@
 	desc = "Some kind of bolt action rifle. You get the feeling you shouldn't have this."
 	icon_state = "moistnugget"
 	inhand_icon_state = "moistnugget"
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'
```

Comment:
handled by build_worn_icon

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon.